<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
   "http://www.w3.org/TR/html4/strict.dtd">
<html lang="en">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
    <meta name="robots" content="noindex,nofollow,noarchive">
<meta http-equiv="Refresh" content="86400">
    <link rel="stylesheet" href="style.css" type="text/css">
    
<script src="saveDataGist.js"></script>
    <title>static feeds</title>
</head>
<body id="rawdog">
<div id="header">
<h1>rawdog</h1>
</div>
<div id="buttons">
<p><form id="token">
    GitHub Token: <input type="password" id="tokenField" autocomplete="on">
    <input type="button" id="tokenButton" onclick="submitToken()" value="Submit">
    
    
  </form></p>
<p><input id="initialButton" type="button" value="Fetch Gist Data" OnClick="getInitial(this.id)"></p>
</div>
<div id="items">
<div class="feeddisplay">
<h3 class="feedtitle"><a id="0"><a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a></a></h3>
<a href="#1">next</a>
<ol class="feedarticles">
<div class="item feed-0320d9ef feed-biodatamininglatestarticles" id="item-cae624c2">
<p class="itemheader">
<span class="itemtitle"><a href="http://www.biodatamining.org/content/6/1/17">Using random walks to identify cancer-associated modules in expression data</a></span>
<span class="itemsave"><input id="button-cae624c2" type="button" onclick="sendPatch('http://www.biodatamining.org/content/6/1/17', this.id)" value="Save"></span>
<span class="itemfrom">[<a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a>]</span>
</p>
<div class="itemdescription">
<p>Background:
The etiology of cancer involves a complex series of genetic and environmental conditions. To better represent and study the intricate genetics of cancer onset and progression, we construct a network of biological interactions to search for groups of genes that compose cancer-related modules. Three cancer expression datasets are investigated to prioritize genes and interactions associated with cancer outcomes. Using a graph-based approach to search for communities of phenotype-related genes in microarray data, we find modules of genes associated with cancer phenotypes in a weighted interaction network.
Results:
We implement Walktrap, a random-walk-based community detection algorithm, to identify biological modules predisposing to tumor growth in 22 hepatocellular carcinoma samples (GSE14520), adenoma development in 32 colorectal cancer samples (GSE8671), and prognosis in 198 breast cancer patients (GSE7390). For each study, we find the best scoring partitions under a maximum cluster size of 200 nodes. Significant modules highlight groups of genes that are functionally related to cancer and show promise as therapeutic targets; these include interactions among transcription factors (SPIB, RPS6KA2 and RPS6KA6), cell-cycle regulatory genes (BRSK1, WEE1 and CDC25C), modulators of the cell-cycle and proliferation (CBLC and IRS2) and genes that regulate and participate in the map-kinase pathway (MAPK9, DUSP1, DUSP9, RIPK2). To assess the performance of Walktrap to find genomic modules (Walktrap-GM), we evaluate our results against other tools recently developed to discover disease modules in biological networks. Compared with other highly cited module-finding tools, jActiveModules and Matisse, Walktrap-GM shows strong performance in the discovery of modules enriched with known cancer genes.
Conclusions:
These results demonstrate that the Walktrap-GM algorithm identifies modules significantly enriched with cancer genes, their joint effects and promising candidate genes. The approach performs well when evaluated against similar tools and smaller overall module size allows for more specific functional annotation and facilitates the interpretation of these modules.
</div>
</div>
<div class="item feed-0320d9ef feed-biodatamininglatestarticles" id="item-71466210">
<p class="itemheader">
<span class="itemtitle"><a href="http://www.biodatamining.org/content/6/1/18">Visualizing genomic information across chromosomes with PhenoGram</a></span>
<span class="itemsave"><input id="button-71466210" type="button" onclick="sendPatch('http://www.biodatamining.org/content/6/1/18', this.id)" value="Save"></span>
<span class="itemfrom">[<a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a>]</span>
</p>
<div class="itemdescription">
<p>Background:
With the abundance of information and analysis results being collected for genetic loci, user-friendly and flexible data visualization approaches can inform and improve the analysis and dissemination of these data. A chromosomal ideogram is an idealized graphic representation of chromosomes. Ideograms can be combined with overlaid points, lines, and/or shapes, to provide summary information from studies of various kinds, such as genome-wide association studies or phenome-wide association studies, coupled with genomic location information. To facilitate visualizing varied data in multiple ways using ideograms, we have developed a flexible software tool called PhenoGram which exists as a web-based tool and also a command-line program.
Results:
With PhenoGram researchers can create chomosomal ideograms annotated with lines in color at specific base-pair locations, or colored base-pair to base-pair regions, with or without other annotation. PhenoGram allows for annotation of chromosomal locations and/or regions with shapes in different colors, gene identifiers, or other text. PhenoGram also allows for creation of plots showing expanded chromosomal locations, providing a way to show results for specific chromosomal regions in greater detail. We have now used PhenoGram to produce a variety of different plots, and provide these as examples herein. These plots include visualization of the genomic coverage of SNPs from a genotyping array, highlighting the chromosomal coverage of imputed SNPs, copy-number variation region coverage, as well as plots similar to the NHGRI GWA Catalog of genome-wide association results.
Conclusions:
PhenoGram is a versatile, user-friendly software tool fostering the exploration and sharing of genomic information. Through visualization of data, researchers can both explore and share complex results, facilitating a greater understanding of these data.
</div>
</div>
</ol></div>
<a id="1"><a href="#0">prev</a></a>
</div>
<h2 id="feedstatsheader">Feeds</h2>
<div id="feedstats">
<table id="feeds">
<tr id="feedsheader">
<th>Feed</th><th>RSS</th><th>Last fetched</th><th>Next fetched after</th>
</tr>

<tr class="feedsrow">
<td><a href="http://www.biodatamining.org/">BioData Mining - Latest Articles</a></td>
<td><a class="xmlbutton" href="http://www.biodatamining.org/latest/rss">XML</a></td>
<td>2013-10-29 14:41</td>
<td>2013-10-30 14:41</td>
</tr>

<tr class="feedsrow">
<td><a href="http://genome.cshlp.org">Genome Research current issue</a></td>
<td><a class="xmlbutton" href="http://genome.cshlp.org/rss/current.xml">XML</a></td>
<td>2013-10-29 14:41</td>
<td>2013-10-30 14:41</td>
</tr>

</table>

</div>
<div id="footer">
<p id="aboutrawdog">Generated by
<a href="http://offog.org/code/rawdog.html">rawdog</a>
version 2.18
by <a href="mailto:ats@offog.org">Adam Sampson</a>.</p>
</div>
<script>
      colorRed('tokenButton');
      loadToken();
      getInitial('initialButton');
    </script>
</body>
</html>
